### PR TITLE
Ft sellers login endpoint

### PR DESCRIPTION
**Pull request by DimejiAre**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works .
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by DimejiAre** [comment]
[API endpoint for sellers login](https://trello.com/c/oeBuQbXn/9-api-endpoint-for-sellers-login)

**Comment by DimejiAre** [comment]
[API endpoint for sellers login](https://trello.com/c/oeBuQbXn/9-api-endpoint-for-sellers-login)

**Comment by DimejiAre** [comment]
[API endpoint for sellers login](https://trello.com/c/oeBuQbXn/9-api-endpoint-for-sellers-login)

**Review by tolls-3 — COMMENTED** [review]
Looks good! 
Note: You may run into some issues with how some files were imported, otherwise, the functionality of the login api endpoint is fine.

File: tests/routes/loginSeller.test.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    expect(res.status).toBe(400)
+    expect(res.body).toEqual({ phone: 'Phone Number is invalid' })
+  })
+  it('checks if user has been logged in successfully with a token being returned', async () => {
+    await request(server)
+      .post('/api/auth/register')
+      .send({
+        phone: '07031900035',
+        password: 'Password12345',
+        password2: 'Password12345'
```
Comment (shaunorpen) [inline tests/routes/loginSeller.test.js]: I thought we were removing password2?

**Review by shaunorpen — APPROVED** [review]
All looks fine to me with only one minor comment (remove password2 from tests), but not anything that means I won't approve :)

**Pull request merged**